src/dynamicprompts/commands/base.py

```diff
@@ -1,5 +1,13 @@
 from __future__ import annotations
 
+from enum import Enum, auto
+
 
 class Command:
     """Base class for commands."""
+
+
+class SamplingMethod(Enum):
```

Comment:
@akx I'm not sure of the best place for the SamplingMethod Enum. What do you suggest?

Reply:
Just `dynamicprompts.enums`? :)